client/pom.xml

```diff
@@ -16,6 +16,9 @@
     <properties>
         <project.mainClass>org.moparforia.client.Launcher</project.mainClass>
         <project.name>Playforia Minigolf Client</project.name>
+        <project.downloadDir>${basedir}/../downloads</project.downloadDir>
```

Comment:
I think download directory should be located in the client `target` directory since this is an artefact used in the package phase.

Reply:
That is indeed a good idea at least as long we only need the JRE for the Client.

Reply:
I would build these platform-specific artefacts only for the client, because otherwise, we will get M x N matrix of artefacts, with at least 3 artefacts (jar, dmg and exe) for each submodule, so we would release at least 12 artefacts for that small project, while in theory, only jars would suffice.

Reply:
Another problem is that currently downloads would not be cleaned with `mvn clean`.

Reply:
I agree and will change it later according to your ideas.